src/app/shared/contact/contact.component.ts

```diff
@@ -0,0 +1,50 @@
+import {Component, OnDestroy, OnInit} from '@angular/core';
+import {FormBuilder, FormControl, FormGroup, Validators} from '@angular/forms';
+import {Subscription} from 'rxjs';
+
+@Component({
+  selector: 'app-contact',
+  templateUrl: './contact.component.html',
+  styleUrls: ['./contact.component.less']
+})
+export class ContactComponent implements OnInit, OnDestroy {
+  contactForm: FormGroup;
+  private subscription: Subscription;
+
+  constructor(private formBuilder: FormBuilder) {
+  }
+
+  ngOnInit(): void {
+    this.initForm();
+    this.formChanges();
+  }
+
+  ngOnDestroy(): void {
+    this.subscription.unsubscribe();
+  }
+
+  initForm(): void {
+    this.contactForm = this.formBuilder.group({
+      name: ['', Validators.required],
+      email: ['', [Validators.required, Validators.email]],
+      topic: ['', Validators.required],
+      message: ['', Validators.required],
+      checkAge: [false]
+    });
+  }
+
+  formChanges(): void {
+    this.subscription = this.contactForm.get('checkAge').valueChanges
```

Comment:
Try to use takeUntil($destroyStream). 
example:
`
private destroyStream = new Subject<void>();

      .pipe(
        takeUntil(this.destroyStream),
      )


  ngOnDestroy(): void {
    this.destroyStream.next();
    this.destroyStream.complete();
  }
`

Reply:
Made unsubscription with a use of takeUntil operator.